damnit/gui/kafka.py

```diff
@@ -38,12 +42,39 @@ def loop(self) -> None:
 
                     self.message.emit(unpickled_msg)
 
+    def run_values_updated(self, proposal, run, name, value):
+        message = msg_dict(MsgKind.run_values_updated,
+                           {
+                               "proposal": proposal,
+                               "run": run,
+                               "values": {
+                                   name: value
+                               }
+                           })
+
+        # Note: the send() function returns a future that we don't await. This
+        # is potentially dangerous, but the reason is because it would cause a
+        # delay in the GUI and because people don't usually close the GUI
+        # immediately after editing a variable so it's unlikely that the process
+        # will die before the update is sent (as happened with the backend).
+        self.kafka_prd.send(self.update_topic, message)
```

Comment:
If it's a concern, I guess one option would be to use `loop.run_soon`, or store a reference to the coroutine and, only when the gui is being closed, await it or `.close()` it?

Reply:
Good idea, I added a call to `.flush()` in fbced4c which should do the same thing.